nucleus/parent/pom.xml

```diff
@@ -678,16 +678,11 @@
             </dependency>
 
             <dependency>
-                <groupId>org.junit.jupiter</groupId>
-                <artifactId>junit-jupiter-engine</artifactId>
+                <groupId>org.junit</groupId>
```

Comment:
I don't like overusing BOMs, because they can cause serious dependency conflicts with other BOMs. Also now you had to declare scope everywhere. Does it have some real advantage? I don't see it.

Reply:
Thank you for review. Fixed. I simply align JUnit in TCK with nucleus parent.